src/content/docs/en/reference/api-reference.mdx

```diff
@@ -2512,19 +2512,17 @@ export const server = {
 
 #### `handler()` property
 
-<p>
-<Since v="4.15.0" />
-</p>
+**Type:** `(input, context) => any`
```

Comment:
This is a nitpicking but shouldn't we conserve the surrounding `<p>` tag? If tomorrow, a default `handler` is added having the `<p>` tag in place could encourage whoever updates the documentation to put the default in the same tag instead of adding a new line or adding another `<p>` tag for default. It's just a thought... nothing mandatory (I'm quibbling).

```suggestion
<p>

**Type:** `(input, context) => any`
</p>
```